Python_level_check_Theoretical_answers.txt

```diff
@@ -35,7 +35,24 @@ What are Decorators?
 - Decorators add functionality to an existing function. 
 
 When a parameter is passed to a function, is it by value (copy)or reference (pointer) ?
-- AFAIK it's by value, i.e. copy.
+- AFAIK it's by value, i.e. copy. `Notes`: Only simple types like int, str, float are passed by value. Collections (list, dict, tuple etc.) are passed by reference. You can check it with the below code:
```

Comment:
Thanks for clarification and memory refreshment 👍 
Will need a deeper recap, though.